pkg/scalers/rabbitmq_scaler.go

```diff
@@ -167,7 +158,7 @@ func (s *rabbitMQScaler) IsActive(ctx context.Context) (bool, error) {
 }
 
 func (s *rabbitMQScaler) getQueueMessages() (int, error) {
-	if s.metadata.includeUnacked {
+	if s.metadata.protocol == httpProtocol {
```

Comment:
With this change, if `http` protocol is used, `unacked` messages are always included and it is no longer optional as it was before. Am I missing anything?

Reply:
Hm, I think this name was confusing. From what I understand the HTTP protocol (`apiHost`) was used only when `includeUnacked==True` otherwise this parameter was unused (amqp case):

> `includeUnacked bool // if true uses HTTP API and requires apiHost, if false uses AMQP and requires host`

Reply:
I've done it, I think it's much more userfriendly to just provide `host` than provide `host` and partial information from it